.npmignore

```diff
@@ -1,4 +0,0 @@
-docs/
-gulp/
-scripts/
-test/
```

Comment:
This is cruft since we are using the `files` key in `package.json`.  No need to ignore when we explicitly include files.